### PR TITLE
LF-4212: Fix "Add another (expense) item" button and adapt it to the new design

### PR DESCRIPTION
**Pull request by Joaquin-M2**
**Description**

The _Add another item_ button disappeared when the user removed the last expense item from a selected expense type and reloaded the page.

The aforementioned UI issue is tackled with the new design for [the new expense items page](https://www.figma.com/design/eMffC0h3JPlWtzNttxJpil/Litefarm-Updated?node-id=91-13211&t=hv5MLnMbMYzxTWFg-0), which requires some changes regarding the _Remove item_ button:
image

In the new design, expense items will have the _Remove item_ button **only if there are two or more** items in the expense type they belong to. Therefore, users won't be able to remove the last item per expense type, which prevents the _Add another item_ button from disappearing. To remove the last item, users will have to unselect the expense type in the previous page:
image

Jira link: https://lite-farm.atlassian.net/browse/LF-4212

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

- [ ] Passes test case
- [x] UI components visually reviewed on desktop view
- [x] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

File: packages/webapp/src/components/Finances/AddExpense/ExpenseItemInputs.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ import { DATE, NOTE, VALUE } from './constants';
 import styles from './styles.module.scss';
 import { useCurrencySymbol } from '../../../containers/hooks/useCurrencySymbol';
 
-export default function ExpenseItemInputs({ register, onRemove, getErrors }) {
+export default function ExpenseItemInputs({ register, onRemove, getErrors, itemsQuantity }) {
```
Comment (SayakaOno) [inline packages/webapp/src/components/Finances/AddExpense/ExpenseItemInputs.jsx:25]: Thank you for clarifying the requirement in the ticket!

Looking at this component as a stand-alone component, it's not clear what `itemsQuantity` refers to. I think something like isRemovable would make more sense.

File: packages/webapp/src/components/Finances/AddExpense/styles.module.scss [hunk 2]
```diff
@@ -51,6 +51,7 @@ p.iconLink {
 .type {
   line-height: 20px;
   margin-bottom: 8px;
+  font-weight: bold;
```
Comment (Duncan-Brain) [inline packages/webapp/src/components/Finances/AddExpense/styles.module.scss:54]: Just curious if this changes the look for you? I see no change.

**Review by Duncan-Brain — DISMISSED** [review]
Looks good to me! 

Only question I had was regarding the style change.

File: packages/webapp/src/components/Finances/AddExpense/styles.module.scss (hunk 2 above)
Comment (Joaquin-M2) [reply packages/webapp/src/components/Finances/AddExpense/styles.module.scss:54]: Where I see this change is on the screenshots at the top of this PR. If you see the first screenshot, the title of the expense type (**Equipment**) has a normal font weight. But in the second screenshot, the title of the expenses (**Equipment** and **Fuel**) have a bold font weight.

Did you use any browser different from Chrome? If so, please let me know which one.

Comment (Duncan-Brain) [reply packages/webapp/src/components/Finances/AddExpense/styles.module.scss:54]: Oh I see it -- I was definitely looking in the wrong spot 😂 . Thanks!

**Review by Duncan-Brain — APPROVED** [review]

**Review by SayakaOno — APPROVED** [review]
Looks good, thank you!

**Pull request merged**